src/eventbridge/send.ts

```diff
@@ -0,0 +1,53 @@
+import { EventBridge } from 'aws-sdk';
+import { EventEntry } from './EventEntry';
+import { Entries } from './Entries';
+import { SendResponse } from './SendResponse';
+import logger from '../observability/logger';
+import { DynamoTestStation } from '../Interfaces/DynamoTestStation';
+
+const eventbridge = new EventBridge();
+const sendModifiedTestStations = async (testStations: DynamoTestStation[]): Promise<SendResponse> => {
+  logger.info('sendModifiedTestStations starting');
+  logger.info(
+    `${testStations.length} test ${testStations.length === 1 ? 'station' : 'stations'} ready to send to eventbridge.`,
+  );
+
+  const sendResponse: SendResponse = {
+    SuccessCount: 0,
+    FailCount: 0,
+  };
+
+  for (let i = 0; i < testStations.length; i++) {
```

Comment:
Nice improvement Prem, but our current linting rules prevent it:
![image](https://user-images.githubusercontent.com/87019731/150813077-96c01946-7dff-40de-9296-7102214b424f.png)

In the little reading I've done on the subject, it seems that this rule is important to follow as we're transpiling because the resulting output is more complex.

Reply:
No worries Andrew, something to look at later on.